nixos/modules/system/boot/loader/raspberrypi/uboot-builder.sh

```diff
@@ -12,7 +12,8 @@ done
 copyForced() {
     local src="$1"
     local dst="$2"
-    cp $src $dst.tmp
+    cp -r $src $dst.tmp
+    rm -rf $dst
```

Comment:
```suggestion
    rm -r $dst
```

we want to know when the file is missing.

Reply:
Do we? When new files gets added here, this `rm` invocation would fail. It will most likely happen whenever a new Raspberry Pi 4 or 5 board gets released.

Reply:
In fact, this PR adds (newly) copying the `overlays/` dir in all cases (maybe worth considering on its own), so if I'm reading right, a `-f` would in fact fail for most people likely.

Reply:
The point is to create `$dst` containing what `$src` contained, regardless of whether `$dst` existed beforehand, so this implementation is exactly correct.